.github/workflows/main.yaml

```diff
@@ -19,7 +19,7 @@ env:
   # go needs absolute directories, using the $HOME variable doesn't work here.
   GOPATH: /home/runner/work/go
 
-  GO_VERSION: '1.21.4'
+  GO_VERSION: '1.22.2'
```

Comment:
We should go to `v1.22.3` directly, given the recent CVEs released for earlier versions: https://groups.google.com/g/golang-announce/c/wkkO4P9stm0